packages/light-apps/src/stores/accountStore.ts

```diff
@@ -0,0 +1,70 @@
+// Copyright 2018-2019 @paritytech/substrate-light-ui authors & contributors
```

Comment:
What is the role of account store? Is it to hold the current account? Is it only to create new accounts? Is it only used in Onboarding or somewhere else?

And do we really need it?
- To know the current account, we could use URLs? i.e. `/address/(identity|transfer)`
- For account creation, we could make the React components self-contained?

Reply:
it is to hold the current account, but I will rewire it so each modal step is self-contained and using URLS instead.